packages/tools/kadena-cli/src/prompts/account.ts

```diff
@@ -306,3 +308,57 @@ export const chainIdPrompt: IPrompt<string> = async (
     },
   })) as ChainId;
 };
+
+function walletKeysToPromptChoices(wallets: IWallet[]): {
```

Comment:
there is a `tableFormatPrompt` method that I've used for aligning values in a prompt, can you check if that can be re-used?

Reply:
it's worked 👍🏽